src/Roots.jl

```diff
@@ -0,0 +1,52 @@
+module Roots
+using ClimaLSM
+using ClimaCore
+import ClimaCore: Fields
+import ClimaLSM:
+    AbstractModel,
+    initialize_prognostic,
+    make_update_aux,
+    make_rhs,
+    make_ode_function,
+    prognostic_vars,
+    auxiliary_vars,
+    initialize_states,
+    initialize_auxiliary
+export RootsModel
+
+"""
+    RootsModel
+
+A standin for the roots model.
+"""
+struct RootsModel{FT, ps} <: AbstractModel{FT}
```

Comment:
not sure if this needs a domain at this stage. can add in when it is clearer!

Reply:
Yes, we can develop this in a follow up PR where we adapt Anna's code to the interface.